Brick-Studios/BeastArena
# Fixed bug where player would get controlled by another player in endgame

# Description

This fixes a bug where when you would hold a player whilst the last level ended and the endgame scene started, the player you held would still be a child(so you would semi control that other player because of relative position of a child). Fixes #53.

# How can this be tested?
Try to reproduce the bug on master and check whether isn't reproducible in this PR.

# Checklist:
